### DISCUSSION TITLE
Python Beginners

### DISCUSSION THREAD

**Discussion by dishant976** [codewars/codewars.com#2947] [General]
Looking for new learners of python, Thanks we can contact in discord

**Comment by uchennaBASH** [comment]
Hello, my name is Uchenna and I am beginner in this programming world. I would love to connect with other beginners as well.

**Reply by dyuan198** [reply]
hello, i am also a beginner, i feel loss here. lol

**Reply by krittyyy19** [reply]
i am also a beginner , can we do it together ?

**Comment by mikefromru** [comment]
Hello, I am beginner too!

**Comment by ElDaldoul** [comment]
Hello, I'd like to join your group. This is my Discord DΛLDΘUL#7084

**Comment by hasnain313khan** [comment]
Hi i am also beginner

**Comment by blu1115** [comment]
I'm new to python id love to join my discord is _blu#1162

**Comment by iamitsfoss** [comment]
Intermediate  python, I'd like to join. 
My Discord is __It's Foss#8268__

**Comment by Aishh-Nair** [comment]
Hello, I am also interested in connecting with everyone here to learn Python and grow. 
ID : Aiswarya#6965

**Comment by 3pleN** [comment]
Hello I am also new at learning python, hope we can help each other :) 
ID:  <blank> #2228

**Comment by caprisunpapy** [comment]
i dont undersatnd why my code is notworking ,on their notebook , therefore it works in my terminal . 

my Python version is 3.10

here is my code : 

def square_all(number:int):
    try:
        result=[]

        str_number=str(number) 

        for i in str_number :
            transform_into_int=int(i) 
            calculus=transform_into_int**2
            result.append(calculus)

        final_result="".join(map(str,result)) 
        final_result=int(final_result)
        print(final_result)

    except TypeError: 
        print("type an integer ")

**Reply by hobovsky** [reply]
Hello,

Please note that Codewars Github discussions are not an appropriate place to discuss coding questions. Github discussions are meant for discussing matters related to the Codewars platform itself. Coding questions can be asked on Codewars Discord server, in channels `#help-solve` or `#python`.

**Comment by LaniKossmannPC** [comment]
Let's keep this a positive and supportive community for beginners to learn and grow together!

**Comment by themoober** [comment]
hello I'm beginner too

**Comment by DawidW10** [comment]
Hello I'm new as well if anyone want to write some code together contact me here or on discord Pandzik#9305. 

Maybe I can help.

**Comment by BurhanxGodhra** [comment]
Hey! i am not new to the coding world but surely want to connect with new people. Lets's jump on my discord burhan.godhra#0217

**Comment by hobovsky** [comment]
If everyone here uses Discord, then maybe let's just get in touch on Codewars Discord? There are channels like `#beginners` and `#python` where everyone can find answers to their questions!
